source/configuration_manual/service_configuration.rst

```diff
@@ -422,9 +423,9 @@ The master auth process. There are 4 types of auth client connections:
 
      This is calculated by counting the process_limit of every service that
      is enabled with the "protocol" setting (e.g. imap, pop3, lmtp).
-     Only services with service_count != 1 are counted, because they have
-     persistent connections to auth, while service_count=1 processes only do
-     short-lived auth connections.
+     Only services with restart_request_count != 1 are counted, because they
+     have persistent connections to auth, while restart_request_count=1
```

Comment:
These could be converted to links here (and elsewhere also).

And BTW for documentation it's not really worth the trouble to keep the lines 80 chars. When writing new text I think it's good to keep them somewhere around 80 chars (or not hugely long), but when modifying existing text it's not really useful.

Reply:
Added links and I also undid the reformatting to reduce the patch size as you suggested.